src/server/pfs/server/testing/server_test.go

```diff
@@ -559,12 +590,12 @@ func TestPFS(suite *testing.T) {
 		require.NoError(t, env.PachClient.PutFile(commit3, "file", strings.NewReader("3")))
 		require.NoError(t, finishCommit(env.PachClient, commit3.Branch.Repo.Name, commit1.Branch.Name, commit3.ID))
 
-		checkRepoCommits := func(commits []*pfs.Commit) {
+		checkRepoCommits := func(repos []string, commits []*pfs.Commit) {
```

Comment:
It might be decent to make this a function taking `[]*pfs.Repo` instead of strings …